src/components/utils/ManagedTree.js

```diff
@@ -46,6 +52,28 @@ let ManagedTree = React.createClass({
     this.setState({ expanded: expanded });
   },
 
+  highlightItem(highlightItems) {
+    const expanded = this.state.expanded;
+
+    // This file is visible, so we highlight it.
+    if (expanded.has(this.props.getKey(highlightItems[0]))) {
+      this.focusItem(highlightItems[0]);
+    } else {
+      let index = highlightItems.length - 1;
+
+      // Look at folders starting from the top-level until finds a
+      // closed folder and highlights this folder
+      highlightItems.some(item => {
```

Comment:
Hi @jasonLaster 
Please check. Thank you! :)
I haven't been able to figure out why I'm failing the test. I couldn't find the error message. Could you help me out with this?